examples/applications/plot_out_of_core_classification.py

```diff
@@ -214,44 +227,67 @@ def progress(blocknum, bs, size):
     'Passive-Aggressive': PassiveAggressiveClassifier(tol=1e-3),
 }
 
+# test data statistics
+test_stats = {'n_test': 0, 'n_test_pos': 0}
 
-def get_minibatch(doc_iter, size, pos_class=positive_class):
+
+def get_test_data(doc_iter_test, pos_class=positive_class):
     """Extract a minibatch of examples, return a tuple X_text, y.
 
     Note: size is before excluding invalid docs with no topics assigned.
 
     """
-    data = [(u'{title}\n\n{body}'.format(**doc), pos_class in doc['topics'])
-            for doc in itertools.islice(doc_iter, size)
-            if doc['topics']]
-    if not len(data):
-        return np.asarray([], dtype=int), np.asarray([], dtype=int)
-    X_text, y = zip(*data)
-    return X_text, np.asarray(y, dtype=int)
 
+    data_test = [(
+        u'{title}\n\n{body}'.format(**doc), pos_class in doc['topics'])
+        for doc in doc_iter_test
+        if doc['topics']]
 
-def iter_minibatches(doc_iter, minibatch_size):
-    """Generator of minibatches."""
-    X_text, y = get_minibatch(doc_iter, minibatch_size)
-    while len(X_text):
-        yield X_text, y
-        X_text, y = get_minibatch(doc_iter, minibatch_size)
+    if not len(data_test):
+        return np.asarray([], dtype=int), np.asarray([], dtype=int)
 
+    X_TextTest, y_test = zip(*data_test)
```

Comment:
please, camel case does not belong here.

Reply:
Greetings!!

Your previous comment (First Comment) was, please use underscores rather than camelCase.

So, to preserve the equality, I do the same.

Thanking you.

Reply:
yes, but you still have camel case here